hudi-common/src/main/java/org/apache/hudi/common/util/collection/BitCaskDiskMap.java

```diff
@@ -61,13 +61,15 @@
  *
  * Inspired by https://github.com/basho/bitcask
  */
-public final class BitCaskDiskMap<T extends Serializable, R extends Serializable> implements DiskMap<T, R> {
+public final class BitCaskDiskMap<T extends Serializable, R extends Serializable> extends DiskMap<T, R> {
 
   public static final int BUFFER_SIZE = 128 * 1024;  // 128 KB
   private static final Logger LOG = LogManager.getLogger(BitCaskDiskMap.class);
   // Caching byte compression/decompression to avoid creating instances for every operation
   private static final ThreadLocal<CompressionHandler> DISK_COMPRESSION_REF =
       ThreadLocal.withInitial(CompressionHandler::new);
+  private static final String BITCASK_DISK_MAP = "bitcask";
```

Comment:
Instead of adding a new string, can't we just reuse the enum w'e have for map type? Like have a getType() abstract method and use it to generate the string?